composer.json

```diff
@@ -16,14 +16,20 @@
             "name": "Walter Tamboer",
             "homepage": "https://waltertamboer.nl",
             "role": "Developer"
+        },
+        {
+            "name": "Mohammad Amin Chitgarha",
+            "email": "machitgarha@outlook.com",
+            "role": "Contributer"
         }
     ],
     "require": {
         "php": "~7.0",
         "ext-gmp": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.6",
+        "php": "~7.2",
```

Comment:
This line seems to conflict with line 27.

I find this a difficult change. On the one hand I'm all in favour of forcing people to newer PHP versions. Especially since 7.1 is EOL. On the other hand, some people are in situations where upgrading simply isn't an option and now those people are prevented from using the library.

Is this neccesary? As long as the library doesn't actually use language features added in >= 7.2 I'd say to not force it.

Reply:
Using PHP 7.2 or greater is only for contributors, not for users; and that is because of using PHPUnit 8. The change is necessary unless you want to downgrade PHPUnit's version.